src/Core/AdminConsole/OrganizationFeatures/Policies/PolicyRequirements/ResetPasswordPolicyRequirement.cs

```diff
@@ -9,24 +10,41 @@ namespace Bit.Core.AdminConsole.OrganizationFeatures.Policies.PolicyRequirements
 public class ResetPasswordPolicyRequirement : IPolicyRequirement
 {
     /// <summary>
-    /// Indicates that new members will automatically be enrolled in account recovery administration when joining an organization.
+    /// List of Organization Ids that require automatic enrollment in password recovery.
     /// </summary>
-    public bool AutoEnrollEnabled { get; init; }
+    public IEnumerable<Guid> AutoEnroll { get; init; }
```

Comment:
I suggest this is private to avoid people misusing it. `AutoEnrollEnabled` is the public interface.

That said, it is still convenient to arrange directly in tests, so maybe there is still a public setter - i.e. a public init property with a private backing field.

```c#
private IEnumerable<Guid> _autoEnrollOrganizations;
public IEnumerable<Guid> AutoEnrollOrganizations { init => _autoEnrollOrganizations = value; }
```